vars/family_RedHat.yml

```diff
@@ -1 +1,2 @@
-aem_dispatcher_module_path: /usr/lib64/httpd/modules
\ No newline at end of file
+aem_dispatcher_module_path: /usr/lib64/httpd/modules
+_aem_dispatcher_apache_server_root: "{{ aem_dispatcher_apache_server_root | default('/etc/httpd') }}"
```

Comment:
See above

Reply:
I don't think that this is necessary. If the dependency can be disabled the apache_server_root will also not be available. The value is set in the same way as in the wcm_io_devops.apache role.

Reply:
Ok, but you're breaking backward compatibility in the sense that you'll never use the value from the dependency even if it is available. Of course you need a static fallback in case the dependency is not available.

Reply:
Ok, regarding the backward compatibility i got your idea. I will check this next week.